README.md

```diff
@@ -101,26 +112,43 @@ Works like [`output.filename`](https://webpack.js.org/configuration/output/#outp
 
 #### `chunkFilename`
 
-Type: `String|Function`
+Type:
+
+```ts
+type chunkFilename =
+  | string
+  | ((pathData: PathData, assetInfo?: AssetInfo) => string);
+```
+
 Default: `based on filename`
 
-> i Specifying `chunkFilename` as a `function` is only available in webpack@5
+> Specifying `chunkFilename` as a `function` is only available in webpack@5
 
 This option determines the name of non-entry chunk files.
 
 Works like [`output.chunkFilename`](https://webpack.js.org/configuration/output/#outputchunkfilename)
 
 #### `ignoreOrder`
 
-Type: `Boolean`
+Type:
+
+```ts
+type ignoreOrder = boolean;
+```
+
 Default: `false`
 
 Remove Order Warnings.
 See [examples](#remove-order-warnings) below for details.
 
 #### `insert`
 
-Type: `String|Function`
+Type:
+
+```ts
+type insert = string | ((linkTag: any) => void);
```

Comment:
Please avoid using `any`, if you found it in our code, it means bug, we should improve this type

Reply:
The linkTag here is an `HTMLElement` right?

Reply:
I don't know, but mostly yes, just check it

Reply:
Changed to `HTMLLinkElement`